src/gas-hub.js

```diff
@@ -33,20 +56,24 @@ $(() => {
           break;
         case 'need relogin':
           initLoginContent();
-          showAlert('Extension has been updated, please relogin', LEVEL_WARN);
+          showLog('Extension has been updated, please relogin', LEVEL_WARN);
           break;
         default:
-          showAlert(err, LEVEL_ERROR);
+          showLog(err, LEVEL_ERROR);
           break;
       }
     });
-});
+}
+
+function unload() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/src/gas-hub.js#L68"></span>‘unload’ is defined but never used.</td>
    </tr>
  </tbody>
</table>